packages/tx-construction/src/tx-builder/initializeTx.ts

```diff
@@ -15,12 +14,11 @@ const dRepPublicKeyHash = async (addressManager?: Bip32Account): Promise<Ed25519
 
 const DREP_REG_REQUIRED_PROTOCOL_VERSION = 10;
 
-const isActive = (drepDelegatee?: Cardano.DRepDelegatee): boolean => {
-  const drep = drepDelegatee?.delegateRepresentative;
-  if (!drep || (Cardano.isDrepInfo(drep) && drep.active === false)) {
-    return false;
-  }
-  return true;
+const hasVoteDelegated = ({ dRepDelegatee }: RewardAccountWithPoolId): boolean => {
+  const drep = dRepDelegatee?.delegateRepresentative;
+  if (!drep) return false;
+  if (!('isDrepInfo' in Cardano)) return true;
```

Comment:
I do not understand this check.

Reply:
Sorry, I should have explain it in a comment.
At the begining this bug had higher prio so it was expected to be merged soon. However the drep info data in the `delegateRepresentative` is available only in the newer core package version provided by the drep provider.
This check attempted to conditionally use drep info data (to check if the drep is active) depending on its availability - minimising required changes on the lace side